cmd/cli/kubectl-kyverno/data/data.go

```diff
@@ -0,0 +1,15 @@
+package data
+
+import (
+	"embed"
+	"io/fs"
+)
+
+const CrdsFolder = "crds"
```

Comment:
Can we re-use `config/crds`? 
If no, we should make sure to update them in codegen as well.

Reply:
We can't, i will definitely automate this.

Reply:
I mean we can't use `embed` with `config/crds`.

Reply:
Done